iocore/net/SSLUtils.cc

```diff
@@ -1895,10 +1963,17 @@ ssl_store_ssl_context(const SSLConfigParams *params, SSLCertLookup *lookup, cons
     keyblock = ssl_context_enable_tickets(ctx, NULL);
   }
 
+  // Generate HPKP header if hpkp is enabled.
+  if (sslMultCertSettings.hpkp_enabled >= 0 ? sslMultCertSettings.hpkp_enabled : params->hpkp_enabled) {
+    hpkp = ssl_context_enable_hpkp(params, sslMultCertSettings);
```

Comment:
It looks like `SSLInitServerContext` doesn't handle SSLCertContext. What I'm missing?